src/test/java/io/pravega/connectors/flink/FlinkTableITCase.java

```diff
@@ -146,6 +155,63 @@ public void testEndToEnd() throws Exception {
         }
     }
 
+
+    /**
+     * Tests the end-to-end functionality of a batch table source & sink.
+     *
+     * <p>This test uses the {@link FlinkPravegaTableSink} to emit an in-memory table
+     * containing sample data as a Pravega stream of 'append' events (i.e. as a changelog).
+     * The test then uses the {@link FlinkPravegaTableSource} to absorb the changelog as a new table.
+     *
+     * <p>Flink's ability to convert POJOs (e.g. {@link SampleRecord}) to/from table rows is also demonstrated.
+     *
+     * <p>Because the source is unbounded, the test must throw an exception to deliberately terminate the job.
+     *
+     * @throws Exception on exception
+     */
+    @Test
+    @Ignore("[issue-124] FlinkPravegaTableSink doesn't support BatchTableSink")
```

Comment:
Instead of ignoring the test case, should we remove and add it back once 124 is done? It can go in the same PR.

Reply:
I would prefer we leave it, since these integration tests are illustrative and we don't have any other samples.   It will be fixed soon in #124